app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/ConfirmationView.scala.html

```diff
@@ -15,22 +15,49 @@
  *@
 
 @import uk.gov.hmrc.nationalimportdutyadjustmentcentrefrontend.views.components.InformationTitle
+@import uk.gov.hmrc.nationalimportdutyadjustmentcentrefrontend.controllers.makeclaim
 @import uk.gov.hmrc.nationalimportdutyadjustmentcentrefrontend.views.html._
 @import uk.gov.hmrc.nationalimportdutyadjustmentcentrefrontend.viewmodels.NavigatorBack
+@import uk.gov.hmrc.nationalimportdutyadjustmentcentrefrontend.models.create.CreateClaimReceipt
 
 @this(
         layout: Layout,
-        formHelper: FormWithCSRF,
+        claimSummaryHeading: CreateClaimSummaryHeading,
+        claimSummary: CreateClaimSummary,
         govukPanel: GovukPanel
 )
 
-@(reference: String)(implicit request: Request[_], messages: Messages)
+@(receipt: CreateClaimReceipt)(implicit request: Request[_], messages: Messages)
+
+@summaryContent = {
+    @claimSummaryHeading(receipt)
+    @claimSummary(receipt.answers, withActions = false)
+}
 
 @layout(pageTitle = InformationTitle(messages("confirmation.title")), NavigatorBack(None)) {
 
+  <div class="nidac-print-hidden">
```

Comment:
Can this be
```suggestion
  <div class="govuk-!-display-none">
```

and the custom styles for `.nidac-print-hidden` removed?

Reply:
Can be if it does the same thing.  I'll check.

Reply:
It doesn't.   `govuk-!-display-none` never displays anything.  What I want is to be able to mark that block so that it displays normally _except_ when printing (when it's not displayed)

Reply:
Sorry, wrong class in my previous comment, i meant `govuk-!-display-none-print`

Reply:
That does the trick!   I'm a dope!  It's exactly the same requirement as hiding back and change links when printing - just on a bigger scale!

Reply:
Or `nidac-!-display-only-print` to be "inspired" by the govuk style naming